pkg/client/cli/cliutil/systema.go

```diff
@@ -127,3 +141,92 @@ func GetCloudLicense(ctx context.Context, outputFile, id string) (string, string
 	}
 	return licenseData.GetLicense(), licenseData.GetHostDomain(), nil
 }
+
+// GetTelepresencePro prompts the user to optionally install Telepresence Pro
+// if it isn't installed. If the user installs it, it also asks the user to
+// automatically update their configuration to use the new binary.
+func GetTelepresencePro(ctx context.Context) error {
+	dir, err := filelocation.AppUserConfigDir(ctx)
+	if err != nil {
+		return errcat.Unknown.Newf("Unable to get path to config files: %s", err)
+	}
+
+	// If telepresence-pro doesn't exist, then we should ask the user
+	// if they want to install it
+	telProLocation := fmt.Sprintf("%s/telepresence-pro", dir)
+	if _, err := os.Stat(telProLocation); os.IsNotExist(err) {
+		reader := bufio.NewReader(os.Stdin)
+		fmt.Printf("Telepresence Pro is recommended when using login features, can Telepresence install it? (y/n)")
+		reply, err := reader.ReadString('\n')
+		if err != nil {
+			return errcat.Unknown.Newf("error reading input: %s", err)
+		}
+
+		// If the user doesn't want to install it, then we we'll proceed
+		// with launching the daemon normally
+		reply = strings.TrimSpace(reply)
+		if reply == "n" {
+			return nil
+		}
```

Comment:
i feel like we should rather return if `reply != "y"` otherwise we risk installing it by accident. This is in line with how eg terraform works

Reply:
ah very smart! ty